### ISSUE TITLE
Add ignoreGetterSetter option for UndocumentedFunction

### ISSUE THREAD

**Issue by arturbosch** [arturbosch/detekt#517]

**Comment by schalkms** [comment]
Detekt's rule `UndocumentedPublicFunction`  does not report the following code snippet for me.
```kotlin
var str: String
  get() = this.toString()
  set(value) { println(value) }
```

**Comment by arturbosch** [comment]
Ah your right, I was thinking about documentation to source code ratios and Java's getter setters ... Closing!
